src/test/scala/org/broadinstitute/dsde/agora/server/webservice/AgoraMethodsSpec.scala

```diff
@@ -107,11 +105,16 @@ class AgoraMethodsSpec extends ApiServiceSpec {
   }
 
   "Agora" should "store 10kb of github markdown as method documentation and return it without alteration" in {
-    Post(ApiUtil.Methods.withLeadingSlash, testAgoraEntityBigDoc) ~>
+    Post(ApiUtil.Methods.withLeadingSlash + "?includedField=documentation", testAgoraEntityBigDoc) ~>
       methodsService.postRoute ~> check {
       handleError(
         entity.as[AgoraEntity],
-        (entity: AgoraEntity) => assert(entity.documentation.get === bigDocumentation.get)
+        (entity: AgoraEntity) => {
+          val searchEntity = new AgoraEntity(name = entity.name, namespace = entity.namespace, snapshotId = entity.snapshotId)
```

Comment:
This change is related to the default projection on findSingle. Are you of the opinion that findSingle should return the entire entity or just the projected entity like find does? In either case I think they should be consistent with each other.